ui/components/multichain/product-tour-popover/product-tour-popover.js

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import Box from '../../ui/box/box';
+import {
+  AlignItems,
+  BLOCK_SIZES,
+  BorderRadius,
+  BackgroundColor,
+  DISPLAY,
+  IconColor,
+  JustifyContent,
+  Size,
+  TextColor,
+  TextVariant,
+  TextAlign,
+} from '../../../helpers/constants/design-system';
+import {
+  ButtonBase,
+  ButtonIcon,
+  IconName,
+  Text,
+} from '../../component-library';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { Menu } from '../../ui/menu';
+
+export const ProductTour = ({
+  className,
+  prevIcon,
+  title,
+  description,
+  currentStep,
+  totalSteps,
+  positionObj = '5%',
+  closeMenu,
+  anchorElement,
+  onClick,
+  prevClick,
+  productTourDirection,
+  ...props
+}) => {
+  const t = useI18nContext();
+  return (
+    <Menu
+      className={classnames(
+        'multichain-product-tour-menu',
+        productTourDirection === 'rtl' && 'multichain-product-tour-menu--rtl',
```

Comment:
This should be:

`'multichain-product-tour-menu--rtl': productTourDirection === 'rtl'`

Reply:
Yesss! Updated here https://github.com/MetaMask/metamask-extension/pull/18571/commits/5a6cab3e9908a72c97c11c268536d279a312543f